api/v1/server/oas/transformers/v1/tasks.go

```diff
@@ -53,6 +53,9 @@ func ToTaskSummary(task *sqlcv1.PopulateTaskRunDataRow) gen.V1TaskSummary {
 			CreatedAt: task.InsertedAt.Time,
 			UpdatedAt: task.InsertedAt.Time,
 		},
+		Input:              jsonToMap(task.Input),
```

Comment:
Input and output really shouldn't be sent in the `TaskSummary` -- can we just remove them from this field and the underlying query? There's a pretty noticeable slowdown if a user has large inputs and outputs.

Reply:
circling back here - I'm pretty sure we need these to render the inputs and outputs on the FE (also: there are a bunch of places where we create an instance of this struct and this is just one of them). are you thinking that we'll get the output dynamically on the FE from the task events? or have a separate query for it? or something else I'm missing?

if you don't think it's dire, I'd prefer to merge this in as-is and then circle back to refactoring this after since this will unbreak some python tests for V1.